project-keeper/src/main/java/com/exasol/projectkeeper/sources/AnalyzedMavenSource.java

```diff
@@ -44,7 +46,7 @@ public static AnalyzedMavenSource.AnalyzedMavenSourceBuilder builder() {
         return new AnalyzedMavenSource.AnalyzedMavenSourceBuilder();
     }
 
-    /** @return source path */
+    /** @return source path, e.g. {@code project-dir/module/pom.xml} */
```

Comment:
:+1:

Reply:
I don't know how often I stumbled across similar code samples in my career and how often I wondered:
* "how could such a path look like?"
* "where is it supposed to start from?"

Reply:
Thanks!